recirq/quantum_chess/circuit_transformer_test.py

```diff
@@ -33,59 +33,64 @@
 @pytest.mark.parametrize('device',
                          (cirq.google.Sycamore23, cirq.google.Sycamore))
 def test_single_qubit_ops(device):
-    transformer = ct.ConnectivityHeuristicCircuitTransformer(device)
     c = cirq.Circuit(cirq.X(a1), cirq.X(a2), cirq.X(a3))
-    transformer.qubit_mapping(c)
-    c = transformer.transform(c)
-    device.validate_circuit(c)
+    t = ct.ConnectivityHeuristicCircuitTransformer(device)
+    device.validate_circuit(t.transform(c))
+    t = ct.DynamicLookAheadHeuristicCircuitTransformer(device)
+    device.validate_circuit(t.transform(c))
```

Comment:
I think we can make pytest parameterize the transformer type for these tests, similar to what we do for the device

Reply:
done!